packages/tempus/src/Tempus_TimeStepControl_decl.hpp

```diff
@@ -129,6 +129,11 @@ class TimeStepControl
       { return tscPL_->get<int>("Number of Time Steps"); }
     virtual Teuchos::RCP<TimeStepControlStrategyComposite<Scalar>>
        getTimeStepControlStrategy() const { return stepControlStrategy_;}
+    virtual int getOutputIndexInterval()
+      { return outputIndexInterval_;}
+    virtual Scalar setOutputTimeInterval()
```

Comment:
I think this should be getOutputTimeInterval().  Do you have tests to cover these additional functions?

Reply:
Not in tempus - where would be the best place to put a test for this in tempus?

Reply:
I just added a TimeStepControl unit test with #6144 (tempus/unit_test/Tempus_UnitTest_TimeStepControl.cpp) (merged this morning).  Add another test to it.